src/test/java/christmas/domain/event/GiveawayEventTest.java

```diff
@@ -0,0 +1,76 @@
+package christmas.domain.event;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import christmas.domain.Amount;
+import christmas.domain.VisitDate;
+import christmas.domain.dish.Dish;
+import christmas.fixtures.ReservationFixtures;
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class GiveawayEventTest {
+
+    private final Event giveawayEvent = new GiveawayEvent(
+            "2023.12.01",
+            "2023.12.25",
+            List.of(new Dish("샴페인", 25_000)));
+
+    private final VisitDate VISIT_DATE = VisitDate.from(4);
+
+    @DisplayName("총 주문 금액이 12만원 이상이면 이벤트가 적용된다")
+    @Test
+    void eventActive() {
+        //given
+        Amount amount = new Amount(13_0000);
+
+        //when
+        Amount giveawayAmount = giveawayEvent.process(
+                VisitDate.from(4),
+                amount,
+                ReservationFixtures.createReservation());
+
+        //then
+        assertThat(giveawayAmount.isEqualTo(25_000)).isEqualTo(true);
```

Comment:
```suggestion
        assertThat(giveawayAmount.isEqualTo(25_000)).isTrue();
```
isTrue나 isFalse를 쓰면 더 깔끔해질 것 같아요